3723/1/page_printer.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'sorted_pages'
+
+class PagePrinter
+  attr_reader :rows, :named_rows
```

Comment:
Not sure that you need public readers here. It's printer, it should print something and maybe return a result from the `call` method. My opinion here is that you can avoid these readers at all. Try to keep public interfaces small and clear. Here you can just use instance variables and get rid of the line